js/enquiry.js

```diff
@@ -19,31 +19,33 @@ $(document).ready(function() {
     $(this).next().text(fileName || 'Upload Files');
   });
 
-  $('#enquiry-form').validate({
-    rules: {
-      services: {
-        required: true,
-        minlength: 1
+  $('form').each(function () {
```

Comment:
The selector is too greedy, we are concatenating all js into one file for deployment. So the selector has to be more specific. Like `form.enquiry`.